python/tests/util/mark.py

```diff
@@ -45,6 +56,27 @@
 Currently controlled by the ARCTICDB_PERSISTENT_STORAGE_TESTS and ARCTICDB_FAST_TESTS_ONLY env vars."""
 
 
+def random_seed_decorator():
```

Comment:
Can't we get rid of a layer of nesting if you make `random_seed_decorator` take an argument `func`, and decorate methods not with `@random_seed_decorator` but with `@random_seed_decorator`?